website/source/docs/operating-a-job/update-strategies/rolling-upgrades.html.md

```diff
@@ -4,73 +4,108 @@ page_title: "Rolling Upgrades - Operating a Job"
 sidebar_current: "docs-operating-a-job-updating-rolling-upgrades"
 description: |-
   In order to update a service while reducing downtime, Nomad provides a
-  built-in mechanism for rolling upgrades. Rolling upgrades allow for a subset
-  of applications to be updated at a time, with a waiting period between to
-  reduce downtime.
+  built-in mechanism for rolling upgrades. Rolling upgrades incrementally
+  transistions jobs between versions and using health check information to
+  reduce down time.
 ---
 
 # Rolling Upgrades
 
-In order to update a service while reducing downtime, Nomad provides a built-in
-mechanism for rolling upgrades. Jobs specify their "update strategy" using the
-`update` block in the job specification as shown here:
+Nomad supports rolling updates as a first class feature. To enable rolling
+updates a job or a specific task group is annotated with a high-level
```

Comment:
nitpick: `a job or a specific task group` -> `a job or task group`

I think `a specific` is implied